Sudo/README.md

```diff
@@ -0,0 +1,33 @@
+# Sudo
+
+Macs with Touch ID have the option to authenticate for sudo using Touch ID instead of the default password authentication.
+
+## Customization
+
+### Manually
+
+This can be enabled manually by editing the relevant PAM file:
+
+```sh
+sudo vim /etc/pam.d/sudo
+```
+
+Add the following to the top of the list:
+
+```sh
+auth       sufficient     pam_tid.so
+```
+
+Press esc, then save and exit
+
+```vim
+:wq!
```

Comment:
Not sure if this is relevant for this section

Reply:
The file is read-only so just ':wq' will throw an error, included vim command to avoid confusion just in case